src/components/App/ExpressRecorder.tsx

```diff
@@ -733,24 +718,43 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
                 <div
                     className={`express-recorder__controls ${styles["express-recorder__controls"]}`}
                 >
-                    {!doRecording && !doCountdown && !doPlayback && (
-                        <button
-                            className={`xr_controls__start ${styles["controls__start"]}`}
-                            id="startRecord"
-                            onClick={this.handleStartClick}
-                            aria-label={this.translator.translate(
-                                "Start Recording. recording will start in a three seconds delay"
-                            )}
-                            tabIndex={0}
-                        />
-                    )}
                     {doRecording && (
                         <RecordingTimer
                             onButtonClick={this.handleStopClick}
                             maxRecordingTime={maxRecordingTime}
                             setStopButtonRef={this.setStopButtonRef}
                         />
                     )}
+                    <div className={styles["settings-wrap"]}>
+                        {!doPlayback && !doRecording && !doCountdown && (
+                            <Settings
+                                selectedCameraDevice={
+                                    stream && stream.getVideoTracks().length > 0
+                                        ? ({
+                                              kind: "videoinput",
+                                              label: stream.getVideoTracks()[0].label
+                                          } as MediaDeviceInfo)
+                                        : false
+                                }
+                                selectedAudioDevice={
+                                    stream && stream.getAudioTracks().length > 0
+                                        ? ({
+                                              kind: "audioinput",
+                                              label: stream.getAudioTracks()[0].label
+                                          } as MediaDeviceInfo)
+                                        : false
+                                }
+                                allowVideo={allowVideo!}
+                                allowAudio={allowAudio!}
```

Comment:
```suggestion
                                allowVideo={allowVideo}
```
why twice?
do not use `!` mark if you are not 100% sure what you are doing.
this mark is risky and might break the application.

Reply:
haha you managed to confuse me.. it's  not twice - it's different props.
I removed the !